packages/headless/src/api/analytics/analytics.ts

```diff
@@ -199,7 +199,7 @@ export const getPageID = () => {
   const lastPageView = actions.reverse().find((action) => {
     return action.name === 'PageView';
   });
-  if (!lastPageView) {
+  if (!lastPageView?.value) {
     return '';
   }
```

Comment:
I'm not too sure what's the context for this PR.

Are you just trying to ignore page views which have no value? If that's the case, then I would suggest the following instead:
```suggestion
  const lastPageView = actions.reverse().find((action) => {
    return action.name === 'PageView' && action.value;
  });
  if (!lastPageView) {
    return '';
  }
```